<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printService</class>
 <widget class="QDialog" name="printService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1199</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1199</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Car Workshop Management</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyświetl zlecenia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń zlecenie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyszukaj zlecenie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj zlecenie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Wyświetl&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Sortuj&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Zmiana na&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Wszystko</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>W trakcie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Do odebrania</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zakończone</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>146</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Brak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imię</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Marka</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numer rejestracyjny</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Status zlecenia</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Brak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>W trakcie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Do odebrania</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zakończone</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Zatwierdź</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Wpisz ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="theme1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>1051</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <zorder>picture</zorder>
  <zorder>theme1</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit</zorder>
  <zorder>tableView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
